compose/src/main/java/com/bottlerocketstudios/compose/snippets/snippetDetails/snippetDetailsWidgets/SnippetDetailsCommentItem.kt

```diff
@@ -0,0 +1,163 @@
+                    modifier = Modifier.padding(top = Dimens.grid_0_5)
+                )
+                Row(
+                    modifier = Modifier.padding(top = Dimens.grid_0_5)
+                ) {
+                    ClickableText(
+                        text = AnnotatedString(stringResource(id = R.string.button_reply)),
+                        style = typography.body1.copy(color = Colors.tertiary),
+                        onClick = { expanded = true }
+                    )
+                    Text(text = "  •  ")
+                    ClickableText(
+                        text = AnnotatedString(stringResource(id = R.string.button_edit)),
+                        style = typography.body1.copy(color = Colors.tertiary),
+                        onClick = { comment.id?.let { it1 -> onEditClick(it1) } }
+                    )
+                    Text(text = "  \u2022  ")
+                    ClickableText(
+                        text = AnnotatedString(stringResource(id = R.string.button_delete)),
+                        style = typography.body1.copy(color = Colors.tertiary),
+                        onClick = { comment.id?.let { it1 -> onDeleteClick(it1) } }
+                    )
+                    Text(text = "  \u2022  ")
+                    Text(
+                        text = comment.updated ?: comment.created ?: "",
+                        style = typography.body2.light().copy(color = Colors.onSurface),
+                        maxLines = 1,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                }
+
+                comment.childrenComments.forEach { comment ->
+                    comment?.let {
+                        CommentCard(
+                            user = user,
+                            replyComment = replyComment,
+                            onReplyChanged = onReplyChanged,
+                            onCancelClicked = onCancelClicked,
+                            comment = comment,
+                            onSaveClick = onSaveClick,
+                            onEditClick = { comment.id?.let { it1 -> onEditClick(it1) } },
+                            onDeleteClick = { comment.id?.let { it1 -> onDeleteClick(it1) } },
+                        )
+                    }
+                }
+            }
+        }
+        if (expanded) {
+            NewCommentInput(
+                user = user,
+                parentId = comment.id,
+                newComment = replyComment,
+                onCommentChanged = onReplyChanged,
+                onSaveClicked = {
+                    expanded = false
+                    onSaveClick(comment.id)
+                },
+                onCancelClicked = {
+                    expanded = false
+                    onCancelClicked()
+                },
+            )
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun PreviewCommentCard() {
```

Comment:
@MMurrayAtBR Make `private`

```suggestion
private fun PreviewCommentCard() {
```